src/videotilecontroller/VideoTileController.ts

```diff
@@ -85,7 +89,9 @@ export default interface VideoTileController {
   addVideoTile(): VideoTile;
 
   /**
-   * Removes a tile by tile id.
+   * Disconnects video source from the video element bounded with the video tile,
```

Comment:
Seeing as this is part of the facade, should we document the purpose of removeVideoTile and what happens when called on local vs remote?
Maybe also discuss why you may want to unbind a video tile vs remove it?

Reply:
I have updated this to tell the difference in the API documentation to make it more clear. Thanks! For the local and remote removeVideoTile acts similar since the removeLocalVideoTile calls removeVideoTile internally with the existing local tileId.